decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,46 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+            <tr>
+              <th><%= t(".user", user_id: "") %></th>
+              <td> <%= user_id = try(:current_user)&.id || t(".unknown") %></td>
+            </tr>
+            <tr>
+              <th><%= t(".date_and_time", date_and_time:"") %></th>
+              <td><%= date_and_time = l(Time.current, format: "%Y-%m-%dT%H:%M:%S.%6N") %></td>
```

Comment:
The assignment of the `date_and_time` variable here is unnecessary as you are not using it anywhere.

Could you move this to the controller and pass it to the view? In fact, you have already created the `info_hash` variable in the controller so maybe you can just pass it to the view and create the table using that?